libs/wingsdk/src/target-tf-aws/app.ts

```diff
@@ -164,16 +166,54 @@ export class App extends CdktfApp {
   /**
    * Returns the VPC for this app. Will create a new VPC if one does not exist.
    */
-  public get vpc(): Vpc {
+  public get vpc(): Vpc | DataAwsVpc {
```

Comment:
nit: Consider returning an interface here that defines the shared functionality of `Vpc` and `DataAwsVpc` (to make the return type Wing-friendly)

Reply:
I did not initially see a shared interface between VPC and DataAwsVpc in cdktf like I would have expected in aws cdk, but let me check again, making this Wing-Friendly is a great point

Reply:
Ah gotcha - yeah if there's no interface provided, you might have to create a new one. If this is internal I'm ok with leaving it as is